sdk/appcenter-kotlin/src/main/java/com/microsoft/appcenter/kotlin/Analytics.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.appcenter.kotlin
+
+import com.microsoft.appcenter.analytics.EventProperties
+import com.microsoft.appcenter.analytics.Analytics as JavaAnalytics
+
+object Analytics : AppCenterService {
+    override val original = JavaAnalytics::class
+
+    suspend fun isEnabled(): Boolean = JavaAnalytics.isEnabled().await()
+
+    suspend fun setEnabled(enabled: Boolean) {
+        JavaAnalytics.setEnabled(enabled).await()
+    }
+
+    fun setTransmissionInterval(seconds: Int): Boolean = JavaAnalytics.setTransmissionInterval(seconds)
+
+    fun pause() = JavaAnalytics.pause()
+
+    fun resume() = JavaAnalytics.pause()
```

Comment:
Looks like a copy-paste error :)

```suggestion
    fun resume() = JavaAnalytics.resume()
```

Reply:
:sweat_smile: